htmd/model.py

```diff
@@ -849,7 +849,11 @@ def plotFES(self, dimX, dimY, temperature, states=False, s=10, cmap=None, plot=T
         s : float
             Marker size for states.
         cmap :
-            Matplotlib colormap.
+            Control both countscmap and statecmap
```

Comment:
I guess it just needs a better doc? `Sets the Matplotlib colormap for both `fescmap` and `statescmap``?

Reply:
ok

Reply:
Writing an example as well